app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -63,15 +63,15 @@ description: |
   is_fedora = @host.operatingsystem.name == 'Fedora'
   os_major = @host.operatingsystem.major.to_i
   os_minor = @host.operatingsystem.minor.to_i
-  realm_compatible = (@host.operatingsystem.name == 'Fedora' && os_major >= 20) || (rhel_compatible && os_major >= 7)
+  realm_compatible = (@host.operatingsystem.name == 'Fedora' && os_major >= 20) || rhel_compatible
```

Comment:
I think we can assume Fedora is also newer than 20 so this would be simplified to
```suggestion
  realm_compatible = is_fedora || rhel_compatible
```

That is effectively:
```suggestion
  realm_compatible = @host.operatingsystem.family == 'Redhat'
```

And we can assume that's the case for kickstart, so really it's
```suggestion
  realm_compatible = true
```

Reply:
Ok, so realm_compatible can be removed at all, as it's used only in one check together with other params, but it would always be true